src/ui/marker.js

```diff
@@ -507,7 +507,12 @@ export default class Marker extends Evented {
             this._pos = this._pos.round();
         }
 
-        DOM.setTransform(this._element, `${anchorTranslate[this._anchor]} translate(${this._pos.x}px, ${this._pos.y}px) ${pitch} ${rotation}`);
+        const el = this._element;
+        const pos = this._pos;
+        const anchor = this._anchor;
```

Comment:
You can get rid of aliases here too (and check for just `this._element`).

Reply:
This is embarrassing, I really thought I removed it!